libs/designer-ui/src/lib/tokenpicker/index.tsx

```diff
@@ -267,7 +267,7 @@ export function TokenPicker({
         ref={containerRef}
       >
         <Nl2fExpressionAssistant
-          tokenGroup={tokenGroup ?? []}
+          tokenGroup={expressionGroup ?? []}
```

Comment:
Does this contain both function expressions and dynamic content ie outputs from previous nodes?

Reply:
good point let me check this.